locale/threadbare.pot

```diff
@@ -0,0 +1,59 @@
+# LANGUAGE translation for Threadbare for the following files:
+# res://scenes/menus/title/components/main_menu.tscn
+# res://scenes/menus/options/options.tscn
+# res://scenes/menus/options/components/sound_settings.tscn
+# res://scenes/menus/options/components/video_settings.tscn
+#
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: Threadbare\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8-bit\n"
+
+#: scenes/menus/title/components/main_menu.tscn
+msgid "Continue"
+msgstr ""
+
+#: scenes/menus/title/components/main_menu.tscn
+msgid "Start"
+msgstr ""
+
+#: scenes/menus/title/components/main_menu.tscn
+msgid "Options"
+msgstr ""
+
+#: scenes/menus/title/components/main_menu.tscn
+msgid "Credits"
+msgstr ""
+
+#: scenes/menus/title/components/main_menu.tscn
+msgid "Exit Game"
+msgstr ""
+
+#: scenes/menus/options/options.tscn
+msgid "< back"
```

Comment:
Oops, that shortcut of putting a `<` into the string rather than an arrow asset is coming back to haunt us!

Reply:
Ha yes and also we have a slightly different one in the Credits scene (capitalized)